server/docker-run-tests.sh

```diff
@@ -0,0 +1,13 @@
+# SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+#!/bin/sh
+
+cat build/configs/database.js
```

Comment:
This path is wrong, it should be `build/src/configs/database.js`. But why is this here?